src/extension/content-script/batteries/YouTubeVideo.ts

```diff
@@ -62,6 +88,40 @@ const battery = async (): Promise<void> => {
   ]);
 };
 
+const battery = async (): Promise<void> => {
+  function waitForDescription() {
```

Comment:
It feels kind of strange that this battery returns a promise but inside sets an interval and returns immediately. Maybe it should have a standard `void` return type?

Reply:
Changing it!